### PR TITLE
Adds boskos and secret config

### PR DESCRIPTION
**Pull request by lwan-wanglin**
Adds boskos config and secret conf for Cluster Profile azure-arm64-qe [ci-tools#3081](https://github.com/openshift/ci-tools/pull/3081)

cc @aleskandro @jianlinliu @MayXuQQ

### CODE REVIEW

**Comment by MayXuQQ** [comment]
need 25 quota slice? at the same time will have 25 jobs ?

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 1]
```diff
@@ -1857,6 +1857,36 @@ secret_configs:
...
+        registry_url: registry.connect.redhat.com
+      - auth_field: auth
+        email_field: email
+        item: registry.redhat.io-pull-secret
+        registry_url: registry.redhat.io
+      - auth_field: token_image-puller_build02_reg_auth_value.txt
+        item: build_farm
+        registry_url: registry.build02.ci.openshift.org
+  to:
+  - cluster_groups:
```
Comment (jianlinliu) [inline core-services/ci-secret-bootstrap/_config.yaml:1886]: I guess we can append this item to the array on line 1855

Comment (lwan-wanglin) [reply core-services/ci-secret-bootstrap/_config.yaml:1886]: thanks, good point. will fix

**Comment by lwan-wanglin** [comment]
> need 20 quota slices? at the same time will have 20 jobs ?

there are no 20 jobs concurrent now, as time and versions grow, and both arm cluster and multi-arch cluster need arm64 vm_type, whether we can keep 20 quota?

**Comment by lwan-wanglin** [comment]
@deepsm007 I am going to add azure ipi for ARM64, I find not all regions have arm64 vm_type, so I added a new Cluster Profile for QE test. I checked Cluster Profile azure4 also contains westus region which doesn't support arm64 vm_type for now.   

Do I need to add a Cluster Profile for Devs CI test  for azure in this PR too?

**Comment by deepsm007** [comment]
> Do I need to add a Cluster Profile for Devs CI test for azure in this PR too?

We should be able to use existing quota from [here](https://github.com/openshift/release/blob/master/core-services/prow/02_config/_boskos.yaml#L550-L609)

**Comment by lwan-wanglin** [comment]
> > Do I need to add a Cluster Profile for Devs CI test for azure in this PR too?
> 
> We should be able to use existing quota from [here](https://github.com/openshift/release/blob/master/core-services/prow/02_config/_boskos.yaml#L550-L609)

if westus--azure-2-quota-slice* is selected, it will hit such issue, 
```
level=error msg=failed to fetch Master Machines: failed to load asset "Install Config": failed to create install config: [controlPlane.platform.azure.type: Invalid value: "Standard_D8ps_v5": not found in region westus, compute[0].platform.azure.type: Invalid value: "Standard_D4ps_v5": not found in region westus]
```

**Comment by deepsm007** [comment]
> westus--azure-2

I didn't realize it was also in there, there are multiple azure profiles and maybe we should add an extra for arm azure with 6 quota-slices including regions except `westus` should be enough. Can you add in the same PR ?

**Comment by lwan-wanglin** [comment]
> > westus--azure-2
> 
> I didn't realize it was also in there, there are multiple azure profiles and maybe we should add an extra for arm azure with 6 quota-slices including regions except `westus` should be enough. Can you add in the same PR ?

how about azure-arm64 profile with quota-slice
```
'azure-arm64-quota-slice': {
        'westus2': 6
    },
```

**Comment by deepsm007** [comment]
> 'azure-arm64-quota-slice': {
>         'westus2': 6
>     },


I believe you mentioned westus doesn't support arm64 vm_type ? It would be better to spread across regions
Which regions support arm64 vm on azure?

**Comment by lwan-wanglin** [comment]
> I believe you mentioned westus doesn't support arm64 vm_type ? It would be better to spread across regions
> Which regions support arm64 vm on azure?

westus is not supported, westus2 is supported. here is all supported regions
```
 az vm list-skus --size Standard_D4ps_v5 --all --output table
ResourceType     Locations       Name              Zones    Restrictions
---------------  --------------  ----------------  -------  --------------
virtualMachines  australiaeast   Standard_D4ps_v5  2,3      None
virtualMachines  centralus       Standard_D4ps_v5  1,2      None
virtualMachines  eastasia        Standard_D4ps_v5  2        None
virtualMachines  eastus          Standard_D4ps_v5  1,2,3    None
virtualMachines  eastus2         Standard_D4ps_v5  3        None
virtualMachines  EastUS2EUAP     Standard_D4ps_v5  2        None
virtualMachines  FranceCentral   Standard_D4ps_v5  3        None
virtualMachines  northeurope     Standard_D4ps_v5  1,2,3    None
virtualMachines  southcentralus  Standard_D4ps_v5  1        None
virtualMachines  southeastasia   Standard_D4ps_v5  1        None
virtualMachines  uksouth         Standard_D4ps_v5  3        None
virtualMachines  westeurope      Standard_D4ps_v5  1,2,3    None
virtualMachines  westus2         Standard_D4ps_v5  1,2,3    None
```

**Comment by lwan-wanglin** [comment]
@jianlinliu could you help to review again? thanks

**Comment by deepsm007** [comment]
/hold 
Remove hold when you feel okay to merge
/lgtm

**Review by MayXuQQ — APPROVED** [review]

**Pull request merged**